tests/constbuffer_thandle_ut/constbuffer_thandle_ut.c

```diff
@@ -1775,7 +1775,7 @@ TEST_FUNCTION(CONSTBUFFER_THANDLE_from_buffer_with_zero_content_succeeds)
     THANDLE_ASSIGN(CONSTBUFFER)(&destination, NULL);
 }
 
-/*Tests_SRS_CONSTBUFFER_THANDLE_88_068: [ If there are any failures then CONSTBUFFER_THANDLE_get_serialized_size shall fail and return CONSTBUFFER_THANDLE_GET_SERIALIZED_SIZE_RESULT_ERROR.]*/
```

Comment:
>Tests_SRS_CONSTBUFFER_THANDLE_88_068 [](http://example.com/codeflow?start=2&length=36)

SRS_CONSTBUFFER_THANDLE_88_068 seems to be not implemented in this unit test. it should be implemented to have full coverage and pass traceability tools also.